### ISSUE TITLE
NuGet packages do not have a package description

### ISSUE THREAD

**Issue by p0utine** [xenial-io/Xenial.Framework#75]
Most Xenial packages have no description set. It comes out as 'Package Description':

![image](https://user-images.githubusercontent.com/87371546/150673247-597fd887-7b94-403b-9b52-359cfedce4bb.png)

We should set a description for each package.

**Comment by biohazard999** [comment]
Yeah we really should add a description that is a little bit more descriptive.
We also should mention what DXVersion that package supports and maybe what the main package is dependend on.

I don't know if package descriptions can have new lines, but it would be nice if we can write an overview with a base line:

```txt
Easy. Flexible. Focused.

Extends DevExpress.ExpressApp (22.2.4)
Provides Code Centric extensions to build Models, DetailViews, ListViews in a refactor safe way
```

or something similar, what do you think @p0utine ?

**Comment by p0utine** [comment]
Good idea. I think new line should work. Apparently it [supports up to 4k characters](https://docs.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices#description). I'll make a few tests and post back.

**Comment by p0utine** [comment]
@biohazard999 It works like a charm! The new line char must be: `%0a`

Base props:
`<Description>Xenial%0aEasy. Flexible. Focused.%0a%0aExtends DevExpress.ExpressApp ($(DxVersion))</Description>`

And we can do something like this in individual packages:
`<Description>%0a$(Description)%0aSpecific description for the package.</Description>`

Giving this:
![image](https://user-images.githubusercontent.com/87371546/152105092-8044ca16-b1be-4f26-b11b-1215e5d94930.png)

If you like it, I will start extracting the sentences from the docs for the individual packages.

**Comment by biohazard999** [comment]
Looks good as proposed!
